vendor/primitives/txpool/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "moonbeam-rpc-primitives-txpool"
-authors = ["PureStake"]
```

Comment:
Not sure we should change this - it was implemented by PureStake.

Reply:
my bad, I missed them when copying vendor files from upstream moonbeam. Below are files only
- `vendor/primitives/txpool/Cargo.toml`
- `vendor/primitives/evm-tracing-events/Cargo.toml`

Reply:
@ashutoshvarma should I add it?

Reply:
Please do, revert the authors and repository from cargo changes, thanks